### PR TITLE
Feat/kafka out only

### PR DESCRIPTION
**Pull request by Tomvbe**

### CODE REVIEW

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/pom.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <artifactId>ldio-connectors</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>ldio-kafka-out</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka</artifactId>
```
Comment (Yalz) [inline ldi-orchestrator/ldio-connectors/ldio-kafka-out/pom.xml:17]: no need to put version? is this also included through the spring pom ?

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+	private String toString(Lang lang, Model model) {
+		return RDFWriter.source(model).lang(lang).build().asString();
+	}
+
+	private Lang getLang(ComponentProperties config) {
+		return config
+				.getOptionalProperty("content-type")
+				.map(RDFLanguages::contentTypeToLang)
+				.orElse(Lang.NQUADS);
```
Comment (Yalz) [inline ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java:40]: Not sure if this is just github or the code itself.. but the spacing seems off, no?

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/pom.xml (hunk 1 above)
Comment (Tomvbe) [reply ldi-orchestrator/ldio-connectors/ldio-kafka-out/pom.xml:17]: yep, not needed

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java (hunk 2 above)
Comment (Tomvbe) [reply ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java:40]: it's github, the block is more to the left in intellij

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+@Configuration
+public class LdioKafkaOutAutoConfig {
+
+	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldio.LdioKafkaOut")
+	public LdioConfigurator ldiKafkaOutConfigurator() {
+		return config -> {
+			final Lang lang = getLang(config);
+			final String topic = config.getProperty("topic");
+			final var kafkaTemplate = createKafkaTemplate(config.getProperty("bootstrap-servers"));
```
Comment (WLefever-Cegeka) [inline ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java:27]: so far, we don't really use the `var` keyword. So, for consistency reasons, I'd suggest `KafkaTemplate<String,String>`

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import java.util.HashMap;
+
+@Configuration
+public class LdioKafkaOutAutoConfig {
+
+	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldio.LdioKafkaOut")
+	public LdioConfigurator ldiKafkaOutConfigurator() {
+		return config -> {
+			final Lang lang = getLang(config);
+			final String topic = config.getProperty("topic");
```
Comment (WLefever-Cegeka) [inline ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java:26]: magic string

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java (hunk 3 above)
Comment (WLefever-Cegeka) [inline ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java:27]: magic string

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+		};
+	}
+
+	private String toString(Lang lang, Model model) {
+		return RDFWriter.source(model).lang(lang).build().asString();
+	}
+
+	private Lang getLang(ComponentProperties config) {
+		return config
+				.getOptionalProperty("content-type")
```
Comment (WLefever-Cegeka) [inline ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java:38]: magic string

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+				.map(RDFLanguages::contentTypeToLang)
+				.orElse(Lang.NQUADS);
+	}
+
+	private KafkaTemplate<String, String> createKafkaTemplate(String bootstrapServer) {
+		final var configProps = new HashMap<String, Object>();
```
Comment (WLefever-Cegeka) [inline ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java:44]: `var` keyword -> Map

**Review by WLefever-Cegeka — APPROVED** [review]
Thanks, Tom. Added some suggestions

File: ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java (hunk 3 above)
Comment (Tomvbe) [reply ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java:27]: This was a topic on a dev meeting that we were  going to allow it if the variable is clear on itself as it improves readability of the codeblock

Comment (Tomvbe) [reply ldi-orchestrator/ldio-connectors/ldio-kafka-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioKafkaOutAutoConfig.java:27]: This was a topic on a dev meeting that we were going to allow it if the variable is clear on itself as it improves readability of the codeblock

**Pull request merged**